### PR TITLE
Various fixes for RHEL8, robot, python3

### PR DESCRIPTION
**Pull request by tgooding**

### CODE REVIEW

File: bb/src/LVUtils.cc [hunk 1]
```diff
@@ -247,7 +247,8 @@ int doInitializeFileSystem(const char* pVolumeGroupName, const char* pDevName, c
                 char* l_DevName = new char[l_Length];
                 snprintf(l_DevName, l_Length, "/%s/%s/%s", DEVICE_DIRECTORY, pVolumeGroupName, pDevName);
                 char l_Cmd[1024] = {'\0'};
-                snprintf(l_Cmd, sizeof(l_Cmd), "mkfs.xfs -f -b log=%d %s 2>&1;", pLogBlockSize, l_DevName);
+//                snprintf(l_Cmd, sizeof(l_Cmd), "mkfs.xfs -f -b log=%d %s 2>&1;", pLogBlockSize, l_DevName);
+                snprintf(l_Cmd, sizeof(l_Cmd), "mkfs.xfs -f -b size=%d %s 2>&1;", 65536, l_DevName);  // \todo fixup
```
Comment (dlherms-ibm) [inline bb/src/LVUtils.cc:251]: Might consider just changing the constant in bbproxy.h...  The comment seems to be already correct...

const int DEFAULT_LOGBLOCKSIZE=16; //Default Block Size of 2**16 is 65536 bytes

File: bb/src/main.cc [hunk 2]
```diff
@@ -90,8 +90,12 @@ uint64_t g_TimeBaseScale;
 
 uint64_t determineTimeBaseScale()
 {
-	uint64_t timebaseScale = 1;
-
+	static uint64_t timebaseScale = 0;
```
Comment (dlherms-ibm) [inline bb/src/main.cc:93]: Don't quite understand this change... The intent is to use g_TimeBaseScale as the global timebase scale factor on bbProxy and bbServer...

File: flightlog/src/flightlog.c [hunk 3]
```diff
@@ -94,6 +94,57 @@ int FL_AttachRegistry(FlightRecorderRegistryList_t** reglist, const char* filena
 	return rc;
 }
 
+static uint64_t getTimeBaseScale()
+{
+	static uint64_t timebaseScale = 0;
```
Comment (dlherms-ibm) [inline flightlog/src/flightlog.c:99]: Similar comment as made in main.cc

File: bb/src/LVUtils.cc (hunk 1 above)
Comment (tgooding) [reply bb/src/LVUtils.cc:251]: The todo comment was to cleanup the LOG references and switch it to bytes through the various function calls.  mkfs.xfs has removed the -b log=<num> syntax.

File: bb/src/main.cc (hunk 2 above)
Comment (tgooding) [reply bb/src/main.cc:93]: We had roughly the same timebase parsing code in 3 places.  This change unifies 2 of those parsers (the 3rd is in flightlog).  Secondly, the 4.18 kernel is apparently 100x slower reading /proc/cpuinfo - not sure yet if this is a config thing on our system, or got a lot more expensive.  This accumulated causing 1.5 seconds of startup time alone and really impacted bbcmd.  The change should to read the value once.

**Pull request merged**